### PR TITLE
Add admin notice for WP 5.6 RC1

### PR DESCRIPTION
**Pull request by nickdaugherty**
## Description

Adds a WP admin notice to publicize the availability of WP 5.6 RC1. 5.6 will be released on December 8th and we encourage all customers to thoroughly test against the latest Release Candidate.

This message will be tweaked prior to deploying this, after we have a Lobby Post announcing the same.

The display conditions for this message are:

* Date is between now and December 9th (day after 5.6)
* WP version is less than 5.6
* User has the `manage_options` capability (which usually means administrator)

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change works and has been tested locally (or has an appropriate fallback).
- [x] This change works and has been tested on a Go sandbox.
- n/a This change has relevant unit tests (if applicable).
- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR
1. Log in as Administrator user
1. Go to `wp-admin` - New notice should display
1. Dismiss notice and reload - should stay dismissed
1. Log in as a non-admin user (Editor)
1. Notice should _not_ display

### CODE REVIEW

File: admin-notice/admin-notice.php [hunk 1]
```diff
@@ -19,12 +19,14 @@
 add_action( 'admin_enqueue_scripts', [ $admin_notice_controller, 'enqueue_scripts' ] );
 add_action( 'wp_ajax_dismiss_vip_notice', [ $admin_notice_controller, 'dismiss_vip_notice' ] );
 
+// WP 5.6 RC1 (released Nov 17, 2020)
 $admin_notice_controller->add(
 	new Admin_Notice(
-		'WordPress 5.5.2 will be released on Friday, October 30th',
+		'WordPress 5.6 will be released on December 8th, 2020. Please ensure your sites have been tested against the current Release Candidate',
```
Comment (mjangda) [inline admin-notice/admin-notice.php:25]: Can we add a link to the Lobby post or something?

Comment (nickdaugherty) [reply admin-notice/admin-notice.php:25]: Yep the previous wording was just a placeholder until the Lobby post went live.

**Comment by nickdaugherty** [comment]
Screen Shot 2020-11-17 at 5 37 52 PM

**Review by brandon-m-skinner — APPROVED** [review]

**Comment by nickdaugherty** [comment]
This was reverted prior to deployment due to a bug with the dismissing of notices.

**Pull request merged**